pkg/config/config_test.go

```diff
@@ -940,3 +941,15 @@ func TestPeerTagsEnv(t *testing.T) {
 	testConfig = SetupConfFromYAML("")
 	require.Equal(t, []string{"aws.s3.bucket", "db.instance", "db.system"}, testConfig.GetStringSlice("apm_config.peer_tags"))
 }
+
+func TestLogDefaults(t *testing.T) {
+
+	c := NewConfig("test", "DD", strings.NewReplacer(".", "_"))
+	require.Equal(t, 0, c.GetInt("log_file_max_rolls"))
+
+	testConfig := SetupConf()
+	require.Equal(t, 1, testConfig.GetInt("log_file_max_rolls"))
```

Comment:
I think we should check all the log keys' defaults here, for both configs.

Reply:
Added more ches.